dev/ses-verify-email.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eo pipefail
 
-EMAIL_ADDRESS=no-reply@asap.yld.io
+EMAIL_ADDRESS=no-reply@hub.asap.science
```

Comment:
Will this (AWS email verification) be a problem? @fampinheiro

Reply:
Email verification for `*@hub.asap.science` is already done.